common/src/main/resources/assets/lighty/lang/en_us.json

```diff
@@ -9,10 +9,16 @@
   "modeSwitcher.lighty.carpet_mode.tooltip": "Original, carpet like overlay",
   "modeSwitcher.lighty.boring_cross_mode": "Boring Cross Mode",
   "modeSwitcher.lighty.boring_cross_mode.tooltip": "It feels like every Light Overlay Mod has them",
+  "modeSwitcher.lighty.farmland_mode": "Farmland Mode",
+  "modeSwitcher.lighty.farmland_mode.tooltip": "Similar to Carpet Mode but exclusively for farmland blocks",
   "modeSwitcher.lighty.settings": "Overlay Settings",
   "settings.lighty.title": "Lighty Settings",
   "lighty.options.block_threshold": "Block Threshold",
   "lighty.options.block_threshold.tooltip": "If the block light level is less than or equal to %d, the overlay is red or orange, depending on the sky light level. Otherwise, it is green.",
+  "lighty.options.farm_growth_threshold": "Farmland Growth Threshold",
+  "lighty.options.farm_growth_threshold.tooltip": "If the block light level is less than or equal to %d, crops won't grow at night and the overlay is orange. Otherwise, it is green.",
+  "lighty.options.farm_uproot_threshold": "Farmland Uproot Threshold",
+  "lighty.options.farm_uproot_threshold.tooltip": "If the block light level is less than or equal to %d, crops will un-plant themselves at night and the overlay is red.",
```

Comment:
I did not add the Portuguese translations. I could take a look but it would be machine-translated...